lang/java/reef-applications/reef-vortex/src/main/java/org/apache/reef/vortex/driver/VortexDriver.java

```diff
@@ -76,7 +79,9 @@ private VortexDriver(final EvaluatorRequestor evaluatorRequestor,
                        @Parameter(VortexMasterConf.WorkerMem.class) final int workerMem,
                        @Parameter(VortexMasterConf.WorkerNum.class) final int workerNum,
                        @Parameter(VortexMasterConf.WorkerCores.class) final int workerCores,
-                       @Parameter(VortexMasterConf.NumberOfVortexStartThreads.class) final int numOfStartThreads) {
+                       @Parameter(VortexMasterConf.NumberOfVortexStartThreads.class) final int numOfStartThreads,
+                       @Parameter(VortexMasterConf.InputCodec.class) final Codec inputCodec,
```

Comment:
You mean more there would be more than one input/output codecs in some applications, right? I agree that it's possible, but I couldn't come up with a clean solution for such case: I thought a map which associates the name of `VortexFunction` with input and output `Codec`, but this was not very pretty. Do you have any suggestion?

I added the parameters to Driver for two reasons: 1) the Driver deserializes the result report in `TaskMessageHandler`, which requires the codec. 2) the codecs should be configured to Workers in `AllocatedEvaluatorHandler`. This would be changed if we have a better solution for configuring the input/output codec including when application wants to more than one.